acquisitions/src/main/resources/thunderjet/mod-consortia/features/user-tenant-associations.feature

```diff
@@ -4,14 +4,484 @@ Feature: Consortia User Tenant associations api tests
+    And params query = queryParams
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)'}
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    And match response.userTenants[0].userId == shadowConsortiaAdminId
+    And match response.userTenants[0].isPrimary == false
+
+    # 6. verify shadow 'consortiaAdmin' has required permissions (in 'universityTenant')
+    * call read(login) universityUser1
+    Given path 'perms/users'
+    And param query = 'userId=' + shadowConsortiaAdminId
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)'}
+    When method GET
+    Then status 200
+
+    And match response.totalRecords == 1
+    And match response.permissionUsers[0].permissions == ['ui-users.editperms']
+
+  Scenario: Verify there are following records for 'consortia-system-user' of 'centralTenant':
+    # 1. 'consortia-system-user' has been saved in 'users' table in 'central_mod_users'
+    * call read(login) consortiaAdmin
+    Given path 'users'
+    And param query = 'username=' + consortiaSystemUserName
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)'}
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    * def consortiaSystemUserInCentralId = response.users[0].id
+
+    # 2. 'consortia-system-user' has been saved in 'user_tenant' table in 'central_mod_users'
+    * call read(login) consortiaAdmin
+    * def queryParams = { username: '#(consortiaSystemUserName)', userId: '#(consortiaSystemUserInCentralId)' }
+    Given path 'user-tenants'
+    And params query = queryParams
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)'}
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    And match response.userTenants[0].tenantId == centralTenant
+
+    # 3. primary affiliation for 'consortia-system-user' has been created in 'user_tenant' table in 'central_mod_consortia'
+    * call read(login) consortiaAdmin
+    * def queryParams = { username: '#(consortiaSystemUserName)', tenantId: '#(centralTenant)' }
+    Given path 'consortia', consortiumId, 'user-tenants'
+    And params query = queryParams
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)'}
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    And match response.userTenants[0].userId == consortiaSystemUserInCentralId
+    And match response.userTenants[0].isPrimary == true
+
+    # 4. 'consortia-system-user' has required permissions
+    * call read(login) consortiaAdmin
+    Given path 'perms/users'
+    And param query = 'userId=' + consortiaSystemUserInCentralId
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)'}
+    When method GET
+    Then status 200
+
+    And match response.totalRecords == 1
+    And match response.permissionUsers[0].permissions == '#[14]'
```

Comment:
What it means #[14] ?

Reply:
and please also insert link to documentation in this PR, maybe in the begining of the test

Reply:
I got it, it is list size, okay it makes sense